lib/language.js

```diff
@@ -5026,6 +5045,7 @@ function init() {
       ,fi: 'Kertoo insuliinin tyypillisen vaikutusajan. Vaihtelee potilaan ja insuliinin tyypin mukaan. Tyypillisesti 3-4 tuntia pumpuissa käytettävällä insuliinilla.'
       ,bg: 'Представя типичната продължителност на действието на инсулина. Варира между отделните пациенти и различни инсулини. Обикновено е 3-4 часа за пациентите с помпа. Нарича се още живот на инсулина '
       ,pl: 'Odzwierciedla czas dzialania insuliny.Może różnić sie w zależnosci od chorego i rodzaju insuliny. Zwykle sa to 3-4 godziny dla insuliny podawanej pompa u wiekszosci chorych. Inna nazwa to czas trwania insuliny.'
+      ,it: 'Rappresenta la tipica duranta di funzionamento dell\'insulina. E 'diverso in ogni paziente e per ogni tipologia d\'insulina. Tipicamente è di 3-4 ore.'
       }
     ,'Insulin to carb ratio (I:C)' : {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Expected an assignment or function call and instead saw an expression.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1574569969/issues/source?bid=2452379&fileBranchId=3022887#l5050)